packages/web-app-user-management/src/index.js

```diff
@@ -9,6 +9,8 @@ function $gettext(msg) {
   return msg
 }
 
+const permissionManager = window.Vue.$permissionManager
```

Comment:
Please move the variable inside both `enabled` callbacks. Reason: safer regarding timing of accessing the `window.Vue.$permissionManager` object. At least for when the `enabled` callbacks are being called we are absolutely sure that the `$permissionManager` instance exists.